app/controllers/declaration/CommodityMeasureController.scala

```diff
@@ -44,15 +44,9 @@ class CommodityMeasureController @Inject()(
   private def form()(implicit request: JourneyRequest[_]) = CommodityMeasure.form(request.declarationType)
 
   def displayPage(mode: Mode, itemId: String): Action[AnyContent] = (authenticate andThen journeyType) { implicit request =>
-    val item = request.cacheModel.itemBy(itemId)
-    val packageInformation = item.map(_.packageInformation)
-    val commodityMeasure = item.flatMap(_.commodityMeasure)
-
-    (packageInformation, commodityMeasure) match {
```

Comment:
Could not work out any reason why PackageInfo was required in order to display CommodityMeasure.  It's been like that for a very long time.